packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed issue where `ProfileInfo.addProfileTypeToSchema` did not update the global schema if a project-level configuration was detected. [#2086](https://github.com/zowe/zowe-cli/issues/2086)
```

Comment:
Can we just add a definition for `ProfileInfo.addProfileTypeToSchema`? Would be better if the note read something like " ... issue where the `ProfileInfo.addProfileTypeToSchema` property/profile/biscuit/whatever did not update ..."

Reply:
Sorry, I should be more specific. I know a similar discussion came up for a changelog in a previous PR.
Thank you for catching this!